### PR TITLE
Update Handwritten Recognition example to keras version3

### PR DESCRIPTION
**Pull request by mehtamansi29**
This example is related to Handwritten Recognition computer vision with keras3 version from keras2.

### CODE REVIEW

**Comment by mehtamansi29** [comment]
> Can you generate the changes in .py file, it would be easier to make the code review.

Sure @sachinprasadhs - I'll do it and commit again.

**Comment by sachinprasadhs** [comment]
Migration to Keras 3 should be the modified changes to the existing file here https://github.com/keras-team/keras-io/blob/master/examples/vision/handwriting_recognition.py and here https://github.com/keras-team/keras-io/blob/master/examples/vision/ipynb/handwriting_recognition.ipynb, no need to create a new file.

**Comment by mehtamansi29** [comment]
> Migration to Keras 3 should be the modified changes to the existing file here https://github.com/keras-team/keras-io/blob/master/examples/vision/handwriting_recognition.py and here https://github.com/keras-team/keras-io/blob/master/examples/vision/ipynb/handwriting_recognition.ipynb, no need to create a new file.

Okay. I'll replace my file with existing one.

File: examples/vision/handwriting_recognition.py [hunk 1]
```diff
@@ -1,10 +1,19 @@
 """
-Title: Handwriting recognition
-Authors: [A_K_Nain](https://twitter.com/A_K_Nain), [Sayak Paul](https://twitter.com/RisingSayak)
-Date created: 2021/08/16
-Last modified: 2023/07/06
-Description: Training a handwriting recognition model with variable-length sequences.
-Accelerator: GPU
+Title: FILLME
```
Comment (fchollet) [inline examples/vision/handwriting_recognition.py:2]: Please fill in these fields

File: examples/vision/handwriting_recognition.py [hunk 2]
```diff
@@ -24,37 +35,48 @@
...
 
 """shell
-wget -q https://github.com/sayakpaul/Handwriting-Recognizer-in-Keras/releases/download/v1.0.0/IAM_Words.zip
-unzip -qq IAM_Words.zip
-
-mkdir data
-mkdir data/words
-tar -xf IAM_Words/words.tgz -C data/words
-mv IAM_Words/words.txt data
+!wget -q
```
Comment (fchollet) [inline examples/vision/handwriting_recognition.py:38]: Shell lines should not be prefixed with `!`

File: examples/vision/handwriting_recognition.py [hunk 3]
```diff
@@ -422,7 +449,7 @@ def build_model():
     # Optimizer.
     opt = keras.optimizers.Adam()
     # Compile the model and return.
-    model.compile(optimizer=opt)
+    model.compile(optimizer=opt, jit_compile=False)
```
Comment (fchollet) [inline examples/vision/handwriting_recognition.py:452]: Why was this necessary?

File: examples/vision/handwriting_recognition.py [hunk 4]
```diff
@@ -240,7 +264,7 @@ def distortion_free_resize(image, img_size):
         ],
     )
 
-    image = tf.transpose(image, perm=[1, 0, 2])
+    image = keras.ops.transpose(image, (1, 0, 2))
```
Comment (fchollet) [inline examples/vision/handwriting_recognition.py:267]: You can just do `from keras import ops` then just use `ops.` instead of `keras.ops.` -- shorter

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: examples/vision/handwriting_recognition.py (hunk 4 above)
Comment (mehtamansi29) [reply examples/vision/handwriting_recognition.py:267]: Hi @fchollet - I am working on all the changes mentioned by you are update with new commit.

**Review by fchollet — APPROVED** [review]
